src/views/TimeOutWrapper/index.tsx

```diff
@@ -1,32 +1,32 @@
 import React, { useEffect } from 'react';
-import { useSelector } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 import { useOktaAuth } from '@okta/okta-react';
-import { DateTime } from 'luxon';
 
+import { updateLastRenewAt } from 'reducers/authReducer';
 import { AppState } from 'reducers/rootReducer';
 
 type TimeOutWrapperProps = {
   children: React.ReactNode;
 };
 
-const sessionTimeout = { minutes: 14 };
-
 const TimeOutWrapper = ({ children }: TimeOutWrapperProps) => {
   const lastActiveAt = useSelector(
     (state: AppState) => state.auth.lastActiveAt
   );
+  const lastRenewAt = useSelector((state: AppState) => state.auth.lastRenewAt);
+  const activeSinceLastRenew = lastActiveAt > lastRenewAt;
```

Comment:
This will always give a user at least one free renew after they've gone inactive., right?

When a user logs in, their `lastActiveAt` is greater than the `lastRenewAt`. If they login and go idle, they'll have the initial 14 minutes, then another 14 minutes from the free renew.

Reply:
Correct. And if we want to make sure that the time is limited to no more than 15 minutes, we can change our Okta settings so that the session window is half the total allowed time. In other words, if I log in, and then immediately go idle, I get an auto-renew at 7 minutes, and then I'm logged out after 14.

Reply:
Currently, the access token appear to be 14 minutes still. I'll approve, and leave it up to you to verify the access token length and have it changed accordingly.